Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -95,10 +96,14 @@ final class PledgePaymentMethodsViewController: UIViewController {
       |> checkoutBackgroundStyle
 
     _ = self.rootStackView
-      |> rootStackViewStyle
+      |> verticalStackViewStyle
```

Comment:
Not sure what the point of this style is, I see we've had it for a while but it just sets the axis to vertical 🤷‍♂

Reply:
Ask @dusi 😆